### PR TITLE
umpv: remove unused imports 

### PR DESCRIPTION
**Pull request by jimman2003**
Self-explanatory i think

### CODE REVIEW

File: TOOLS/umpv [hunk 1]
```diff
@@ -31,9 +31,6 @@ import os
 import socket
 import errno
 import subprocess
-import fcntl
-import stat
-import string
```
Comment (sfan5) [inline TOOLS/umpv:36]: Line 46: `    allowed_symbols = string.ascii_letters + string.digits + '_'`

Comment (jimman2003) [reply TOOLS/umpv:36]: The weird thing is that it ran without exceptions!..i will fix that and squash..

**Review by jimman2003 — COMMENTED** [review]
Done!

File: TOOLS/umpv (hunk 1 above)
Comment (jimman2003) [reply TOOLS/umpv:36]: Who about now?

**Comment by jimman2003** [comment]
Forgot to sync! I think it is ready now..

**Comment by jeeb** [comment]
I'll just double-check and pull it in unless there's something really surprising.

**Comment by jeeb** [comment]
Change-wise looks good to me:

* fcntl usage was replaced by socket usage in 518bd4c306d50e6772c39c5d7395b9d10b9386da
* stat usage was removed in 51a3f13705f8b65b3bfcef5b991903d225759014 as the socket was created in the user's HOME.

**Comment by jeeb** [comment]
@jimman2003 is the authorship information just as you wish it to be?

**Comment by jeeb** [comment]
Just reworded the commit a bit (https://github.com/mpv-player/mpv/commit/ceec442fe13cd661b977c229692384c669d9b9dc) , otherwise OK, so other than this verification it's good to be pulled in.

Just some notes for the future:
1. The "rebase and merge" function of Github generally will take care of updating, the only case where this isn't possible is when there's a merge conflict, which for this file is unlikely. In those cases don't merge, but instead rebase.
2. Do note the slight updates to the commit message, such as having a space after the prefix and small initial letter. Proper writing then in the following paragraphs. I just added the comment on where those specific things' usage was dropped before.

**Comment by jimman2003** [comment]
I want to attibute myself as an author... No clue how. Otherwise I give my consent to include my change with all licenses that apply in this file!

**Comment by jeeb** [comment]
`git commit --amend --author="Name Surname <email@address.tld>"`

**Comment by jeeb** [comment]
But if you are not sure how to do it, just note how you want to be mentioned in the commit and I can do the update since I've already updated the commit message :)

**Review by jeeb — APPROVED** [review]
LGTM

**Pull request merged**